src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -112,13 +113,16 @@ public bool Equals(HostAssignmentContext other)
             return SiteId == other.SiteId && LastModifiedTime.CompareTo(other.LastModifiedTime) == 0;
         }
 
-        public void ApplyAppSettings(IEnvironment environment)
+        public void ApplyAppSettings(IEnvironment environment, ILogger logger)
         {
             foreach (var pair in Environment)
             {
                 environment.SetEnvironmentVariable(pair.Key, pair.Value);
+                if (pair.Key.StartsWith("WEBSITE_AUTH"))
+                {
+                    logger.LogWarning($"ApplyAppSettings {pair.Key} = {pair.Value}");
```

Comment:
is this ok to log? What are the values? We should not be logging any secrets. Also change to logdebug every where instead of warning.